tests/main/interfaces-many-core-provided/task.yaml

```diff
@@ -55,6 +54,16 @@ execute: |
     echo "For each core-provided slot"
     SNAP_MOUNT_DIR="$(os.paths snap-mount-dir)"
     for plugcmd in "$SNAP_MOUNT_DIR"/bin/"$CONSUMER_SNAP".* ; do
+
+        # Just connect 30% of the interfaces on debian 10
+        # Debian 10 has bad performance disconnecting interfaces
+        # and the test fails (kill-timeout) trying either to remove
+        # interfaces or removing the snap
+        if os.query is-debian-10 && [ "$((RANDOM % 3))" = 0 ]; then
```

Comment:
Sorry, I think I made a mistake. This should be `.. != 0` as we're skipping if the condition is true.

Reply:
You are right, updated